modules/stanford_soe_helper_homepage/stanford_soe_helper_homepage.info

```diff
@@ -0,0 +1,19 @@
+name = Stanford SoE Helper Homepage
+description = All things related to the SoE homepage
+core = 7.x
+package = Stanford
+version = 7.x-1.0
+project = stanford_soe_helper_homepage
+dependencies[] = bean
+dependencies[] = context
+dependencies[] = ctools
+dependencies[] = stanford_events_importer
+dependencies[] = stanford_events_views
+dependencies[] = stanford_news_extras
+dependencies[] = stanford_news_extras_views
+dependencies[] = views
+features[context][] = stanford_jumpstart_soe_home
+features[ctools][] = context:context:3
+features[features_api][] = api:2
+project path = sites/default/modules/stanford/stanford_soe_helper/modules
```

Comment:
Please remove project path so when the next person exports this module it goes to the correct place.